test/metabase/task/delete_inactive_tables_test.clj

```diff
@@ -12,21 +12,30 @@
         older-than-threshold        (t/minus (t/zoned-date-time)
                                              (t/days (inc @#'delete-inactive-tables/inactive-table-max-days)))]
     (mt/with-temp
-      [:model/Database {db-id :id}                 {}
-       :model/Table    {new-active-table-id :id}   {:db_id       db-id
-                                                    :active      true
-                                                    :updated_at  one-day-less-than-threshold}
-       :model/Table    {old-active-table-id :id}   {:db_id       db-id
-                                                    :active      true
-                                                    :updated_at  older-than-threshold}
-       :model/Table    {new-inactive-table-id :id} {:db_id       db-id
-                                                    :active      false
-                                                    :updated_at  one-day-less-than-threshold}
-       :model/Table    _old-inactive-table         {:db_id       db-id
-                                                    :active      false
-                                                    :updated_at  older-than-threshold}]
+      [:model/Database         {db-id :id}                             {}
+       :model/Table            {new-active-table-id :id}               {:db_id       db-id
+                                                                        :active      true
+                                                                        :updated_at  one-day-less-than-threshold}
+       :model/Table            {old-active-table-id :id}               {:db_id       db-id
+                                                                        :active      true
+                                                                        :updated_at  older-than-threshold}
+       :model/Table            {new-inactive-table-id :id}             {:db_id       db-id
+                                                                        :active      false
+                                                                        :updated_at  one-day-less-than-threshold}
+       :model/Table            {old-inactive-table-id :id}             {:db_id       db-id
+                                                                        :active      false
+                                                                        :updated_at  older-than-threshold}
+       :model/Card             {_inactive-card :id}                    {:database_id db-id
+                                                                        :archived    true
+                                                                        :table_id    old-inactive-table-id}
+       :model/Table            {old-inactive-table-active-card-id :id} {:db_id       db-id
```

Comment:
This confused me a little, since it reads like a card ID
```suggestion
       :model/Table            {old-inactive-table-active-with-card-id :id} {:db_id       db-id
```